### ISSUE TITLE
TypeError: '>' not supported between instances of 'str' and 'int'

### ISSUE THREAD

**Issue by maxandersen** [OblivionCloudControl/opvault#6]
running 1opvault-cli my.opvault "name"1 gives me:

```
1Password master password:
Traceback (most recent call last):
  File "/usr/local/bin/opvault-cli", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/opvault/cli.py", line 83, in main
    vault.unlock(master_password=master_password)
  File "/usr/local/lib/python3.7/site-packages/opvault/onepass.py", line 121, in unlock
    salt = bytes(self.decode_function(self._profile_json['salt'].encode()))
  File "/usr/local/lib/python3.7/site-packages/opvault/onepass.py", line 113, in decode_function
    if self.is_python31_or_newer():
  File "/usr/local/lib/python3.7/site-packages/opvault/onepass.py", line 109, in is_python31_or_newer
    return bool((major == 3 and minor > 1) or major > 3)
TypeError: '>' not supported between instances of 'str' and 'int'
```

**Comment by bppdddqqqq** [comment]
It seems that the way the function handles version matching is botched in later releases since Python 3.1.

Removing the contents of the function and merely replacing it with `return True`, remedies the problem :+1:

**Comment by steynovich** [comment]
Just released version 0.4.7 which fixes this issue
